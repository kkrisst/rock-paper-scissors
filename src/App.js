import React from 'react';

import { BrowserRouter, Switch, Route } from 'react-router-dom';

import MenuPage from './pages/menu-page/menu-page.component.jsx';

import './App.scss';

function App() {
  return (
    <div className="App">
      <BrowserRouter>
        <Switch>
          <Route exact path='/' component={MenuPage} />
        </Switch>
      </BrowserRouter>
    </div>
  );
}

export default App;
