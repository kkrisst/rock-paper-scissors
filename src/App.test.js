import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme'

import App from './App';

describe('Testing the App component', () => {

  const wrapper = shallow(<App />);

  it('expect MenuPage to be defined', () => {
    expect(App).toBeDefined();
  });

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<App />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('expect MenuPage to match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

})