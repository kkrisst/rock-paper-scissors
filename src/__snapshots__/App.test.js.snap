// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the App component expect MenuPage to match snapshot 1`] = `ShallowWrapper {}`;
